src/include/azure_extension.hpp

```diff
@@ -113,6 +113,11 @@ class AzureStorageFileSystem : public FileSystem {
 
 	static void Verify();
 
+public:
+	static mutex azure_log_lock;
```

Comment:
nit: Maybe add a small comment here to explain to the reader why there are global variables here